Tests/MapboxCoreNavigationTests/NavigationServiceTests.swift

```diff
@@ -59,10 +58,6 @@ class NavigationServiceTests: XCTestCase {
         try? Navigator.shared.navigator.resetRideSession()
     }
 
-    func testDefaultUserInterfaceUsage() {
-        XCTAssertTrue(!dependencies.navigationService.eventsManager.usesDefaultUserInterface, "MapboxCoreNavigationTests shouldn't have an implicit dependency on MapboxNavigation due to removing the Example application target as the test host.")
```

Comment:
There should still be a test that `usesDefaultUserInterface` is false when the developer installs Core Navigation alone.

Reply:
Now the `TestHelper` has the dependency of both `MapboxNavigation` and `MapboxCoreNavigation`. when it's running under SPM, it would be true. When it's under Carthage, because we removed the Example app as the host, it would be false. After the fix of #2922 , it may be fixed.